schema.sql

```diff
@@ -38,4 +38,22 @@ CREATE TABLE visits (
     visit_date DATE,
     vets_id BIGINT REFERENCES vets(id),
     animals_id BIGINT REFERENCES animals(id)
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+
+-- ---------------------------
+-- ALTER TABLE visits ADD COLUMN id BIGSERIAL PRIMARY KEY; 
+
+-- Optimize first querie
+
+CREATE INDEX asc_animals ON visits(animals_id ASC);
+
+-- Optimize second querie
+
+CREATE INDEX asc_vets ON visits(vets_id ASC);
+
+-- Optimize third querie
+
+CREATE INDEX asc_emails ON owners(email ASC);
```

Comment:
- [ ] You were expected to include 3 pairs of screenshots in the PR description with the results of the  EXPLAIN ANALYZE, the pairs should be in `before` and `after` pair in order to read them so easily, kindly address this and request for another review